docs/index.md

```diff
@@ -90,6 +92,13 @@ const api = new RippleAPI({
 api.on('error', (errorCode, errorMessage) => {
   console.log(errorCode + ': ' + errorMessage);
 });
+api.on('connected', () => {
+  console.log('connected');
+});
+api.on('disconnected', (onError) => {
+  // onError will be true if disconnected due to error
```

Comment:
Also, `onError` is the name of a function, if it's a boolean it should be `isError` or something